### PR TITLE
Update code by solving linter issues.

### PR DESCRIPTION
**Pull request by splattater**

### CODE REVIEW

File: ChangesetDiff.py [hunk 1]
```diff
@@ -6,11 +6,15 @@
...
+    """Serialize RDF deltas using the Changeset vocabulary.
+
+    More info:
+    * http://vocab.org/changeset/schema.html
+    """
 
+    def serialize(self, add, delete):
 
         changeset = Namespace("http://purl.org/vocab/changeset/schema#")
-        #@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
```
Comment (white-gecko) [inline ChangesetDiff.py:13]: Do we need this out commented line?

File: ChangesetDiff.py [hunk 2]
```diff
@@ -23,30 +27,31 @@ def serialize(self, add, delete):
...
+                        g.add((graphTerm, changeset.addition, insertGraphName))
+                        g.add((insertGraphName, RDF.type, RDF.Statement))
+                        g.add((insertGraphName, RDF.subject, triple[0]))
+                        g.add((insertGraphName, RDF.predicate, triple[1]))
+                        g.add((insertGraphName, RDF.object, triple[2]))
                         i += 1
 
         return g.serialize(format="turtle").decode("utf-8")
 
+
```
Comment (white-gecko) [inline ChangesetDiff.py:56]: Do we need this extra blank line?

File: EccrevDiff.py [hunk 3]
```diff
@@ -6,39 +6,44 @@
...
+
+    More info:
+    * https://github.com/eccenca/eccrev-vocab
+    """
 
+    def serialize(self, add, delete):
 
         commit = Namespace("urn:commit:" + str(uuid.uuid1()) + ":")
         eccrev = Namespace("https://vocab.eccenca.com/revision/")
-        #@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
```
Comment (white-gecko) [inline EccrevDiff.py:14]: Do we need this out commented line?

File: EccrevDiff.py [hunk 4]
```diff
@@ -6,39 +6,44 @@
...
                 if graphUri in add.keys() and len(add[graphUri]) > 0:
                     insertGraphName = revision.term(":insert")
-                    g.add( (revision.term(""), eccrev.deltaInsert, insertGraphName) )
+                    g.add((revision.term(""), eccrev.deltaInsert, insertGraphName))
                     for triple in add[graphUri]:
                         g.add(triple + (insertGraphName,))
 
         return g.serialize(format="trig").decode("utf-8")
 
+
```
Comment (white-gecko) [inline EccrevDiff.py:48]: do wee need this empty line?

**Review by white-gecko — CHANGES_REQUESTED** [review]

File: ChangesetDiff.py (hunk 2 above)
Comment (splattater) [reply ChangesetDiff.py:56]: In some cases, yes:
`E305 expected 2 blank lines after class or function definition, found 1 [pep8]`

File: ChangesetDiff.py (hunk 1 above)
Comment (splattater) [reply ChangesetDiff.py:13]: no ;-)

Comment (white-gecko) [reply ChangesetDiff.py:13]: sorry

**Review by white-gecko — APPROVED** [review]
Everything is fine

**Review by white-gecko — APPROVED** [review]

**Pull request merged**